src/app/episode/info/store/info.actions.ts

```diff
@@ -3,5 +3,5 @@ import { EpisodeInfo } from '../../episode.models';
 
 export const changeInfoPropertiesAction = createAction(
   '[Info] Change Properites',
-  props<Partial<EpisodeInfo>>()
+  props<Partial<EpisodeInfo>>(),
```

Comment:
This need to be refactored. Redux will add property `type` to the payload. Resulting an EpisodeInfo JSON with an `type: "[Info] Change Properties"` in the end.

To fix it, the parameter must never be an object with first-class citizens, but something like:
`props<{ episodeInfo: Partial<EpisodeInfo>}>`

Reply:
Is this something that we should create an issue for? Seems like something worth capturing. Or did you want me to resolve it on this branch before we complete the PR?

Reply:
Let's create a new bug, it can be fixed and refactored downstream, too much stuff in this PR already

Reply:
Tracking this issue on #61